src/BenchmarkDotNet.Core/Engines/ConsoleHostApi.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+using JetBrains.Annotations;
+
+namespace BenchmarkDotNet.Engines
+{
+    public sealed class ConsoleHostApi: IHostApi
+    {
+        private readonly TextWriter _outWriter;
```

Comment:
In BenchmarkDotNet, we don't use `_` as the private field prefix. Please, remove it.